subsys/logging/Kconfig.backends

```diff
@@ -12,6 +12,10 @@ config LOG_BACKEND_UART
 
 if LOG_BACKEND_UART
 
+config LOG_BACKEND_UART_ASYNC
+	bool "Use UART Asynchronous API"
+	depends on UART_ASYNC_API
```

Comment:
May also add `!CONFIG_LOG_BACKEND_UART_OUTPUT_DICTIONARY_HEX` as dictionary output in hex is still using polling for output.

Reply:
done